randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -314,7 +314,7 @@ Extra - asset_id: collision_camera_006
   > Door to Cold Room (Small)
       Any of the following:
           Speed Booster
-          Movement (Beginner) and Lay Cross Bomb
+          Cross Bomb Launch (Beginner) and Movement (Beginner) and Lay Cross Bomb
```

Comment:
Should not include beginner movement.

Reply:
The reason I left the movement in these is that you still need to jump out, which is not the easiest thing in the world, but maybe people's mileage may wary.